attestation-evidence-broker/src/aeb_modules/sev_aeb/get_evidence.rs

```diff
@@ -0,0 +1,144 @@
+extern crate serde;
+extern crate sev;
+
+use self::sev::certs::ca;
+use self::sev::certs::sev::{Certificate, Chain, Usage};
+use self::sev::firmware::{AttestationReport, Firmware};
+use self::sev::Generation;
+
+use anyhow::*;
+use codicon::Decoder;
+use rand::Rng;
+use std::io::{Error, ErrorKind};
+use std::time::Duration;
+
+use crate::aeb_modules::sev_aeb::evidence::SevEvidence;
+
+fn download(url: &str, usage: Usage) -> Result<Certificate> {
+    let mut rsp = reqwest::blocking::get(url);
+    let mut http_request_replies = Vec::new();
+    for request_wait_seconds in &[0, 2, 4, 6, 8] {
```

Comment:
Can you add a comment to explain why the five attempts of 0, 2, 4, 6 and 8 are adopted here?